### ISSUE TITLE
Incognito payouts

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#3422]
## User story 
As an expense submitter, I want to have the option to protect my privacy, while still having the budget be transparent.

## Best solution for this problem
Incognito option for payouts, similar to incognito for financial contributions - the pay to profile displays 'incognito' but the admins can still see the actual submitter profile info. This would be a tickbox option when submitting an expense. This way people will still be able to see what the budget is paying for but not necessarily who.

## MVP
Right now you can approximate this by creating a pseudonym profile, but most users don't know about this and it's not easy for them. I can't think of an MVP option that would as straightforward and accessible as incognito.

This is important for Collectives supporting sensitive things, which are a growing demographic (e.g. mutual aid groups, a family event group we just had sign up in NZ).

**Comment by Betree** [comment]
We could make the `Incognito` profile available in the `Payee` select. The main issue with that is about tax form, we would need to consider the incognito profile when checking for the tax form status of a user.

**Comment by alanna** [comment]
This is becoming more important because we have a lot of mutual aid groups in Foundation funding sensitive expenses.

**Comment by kaylarep** [comment]
+1 for this to be implemented. 

Even just making a clear distinction labeling the input boxes as:
'Display Name' (This will be publicly displayed)
and then a place for 'Legal Name' (For Tax Purposes)

**Comment by elea** [comment]
+1 – this recently came up for our project when a trans payee thought they had to input their legal / dead name for sign-up (because they’re only on the platform to submit invoices/expenses), and now the expenses list is displaying their dead name publicly.

**Comment by znarf** [comment]
Problem, some people are expecting the opposite and based on their feedback, we had to make sure this is the way it is today.

**Comment by alanna** [comment]
> some people are expecting the opposite

Can you expand on this @znarf ?

Obviously some Collectives will not want incognito payouts. But in that case, they can just reject such expenses or tell the submitter they need to reveal their name to be paid. And Collectives that need privacy can allow it.

**Comment by znarf** [comment]
@alanna posted the comment on the wrong issue, sorry for the confusion. will hide them ...

**Comment by alanna** [comment]
Bumping this again as we've received more user requests for it.

**Comment by patcon** [comment]
+1 We have a mutual aid group needing this, and we'd mistakenly reassured them this was workable.

Is there a recommended manual workaround? My (perhaps incorrect) impression is that the simplest change for creating a viable work-around is:
1. Direct invoicing users (either invited or existing) to leave public profile anonymous, and
2. Ensure they include full name in banking details, which will only appear on private invoice. #todo Needs this/these field(s) added.

It might not always be correct (business bank account?), but it will be better than forcing everyone to doxx themselves to cover backend accounting requirements :)

Still def room for improvement in UX, but can't see a downside to including some extra info on invoices

**Comment by alanna** [comment]
@patcon that's correct. They can use a pseudonym on their profile and their real name for the payout details/invoice, which will not be public.

We need a better solution for this situation though, because it's in tension with #3340

**Comment by patcon** [comment]
> **real name for the payout details**/invoice, which will not be public

Which, to confirm, "real name for the payout details" will currently be missing from invoice PDF right now. So it's not legally compliant as an accounting record.

Screenshot of :lock: [this invoice](https://opencollective.com/equity4who/expenses/41034):

<img width="717" alt="Screen Shot 2021-05-27 at 12 50 29 AM" src="https://user-images.githubusercontent.com/305339/119767856-b0bc7280-be85-11eb-80d6-9b8e0808c91d.png">

**Comment by alanna** [comment]
@BenJam - in terms of roadmap priorisation, I think this would fall under 'fundmentals'. We have to have valid invoices for accounting purposes, but the privacy workaround is breaking that, and enforcing real names on profiles would break important privacy needs. Since we already have the architecture for incognito contributions, is applying it to payouts particularly complex?

**Comment by alanna** [comment]
We now have better support for pseudonyms with the display/legal name settings. However, I still think incognito payouts would be a good feature.

**Comment by Betree** [comment]
@alanna I indeed think we have all we need with this new concept of "legal name"/"display name" to implement this feature. In the backend. Having an incognito concept at the expense level (some expenses being incognito and others not) is desirable, and could be achieved quite easily by allowing users to submit expenses with their incognito profile (we would still use the legal name from their user profile).

Since the payout information and legal names are already private and protected, that should be good for an MVP. So all that's left to do before we can tackle this issue is an input from @opencollective/design to define how that will look in the expense flow.

**Comment by aerugo** [comment]
+1 for this

I think this will be key in getting OC implemented for more traditional orgs.

**Comment by BenJam** [comment]
given the conversations happening elsewhere (#5270) and the fact that we have protections in place now for users who are concerned about their public perceptions and safety I dont think we should actively deprioritise this. 

Giving orgs the choice to use OC without having to be more transparent seems pretty much against the philosophy of a platform that breeds trust in communities by being more transparent.

**Comment by aerugo** [comment]
EDIT: Sorry, wrote Cobudget instead of OpenCollective. Changed. ;)

**Comment by alanna** [comment]
Let's keep discussing this here on this issue. I think good points are being raised on both sides. Thanks for your feedback @aerugo - it's very useful to get different perspectives.

Right now we offer some partial solutions or workarounds, including using a pseudonymous display name (although this still allows people who know who works on the project to guess who is who), or manually aggregating payouts to a group profile instead of breaking them down individually (although this diminishes the operational efficiency of OC by not automating payments).

I genuinely feel torn between supporting functionality that some users want (which is clear from the use of workarounds in practice), and using our design decisions to push people toward more transparency as an ideological stance.

Regardless of any of that, this feature is not currently being prioritised because there are other more pressing issues we need to work on. So we can keep exploring ideas.

**Comment by Betree** [comment]
Cross-referencing the comment from @agnostic-apollo, who made some interesting suggestions about an "Expense privacy control" step in the expense flow: https://github.com/opencollective/opencollective/issues/4729#issuecomment-1199264466
